### ISSUE TITLE
Adapter does not synchronize new objects from deconz anymore

### ISSUE THREAD

**Issue by psanwald** [Jey-Cee/ioBroker.deconz#184]
First of all: Thank you for this adapter - this is kind of a live saver for me! My problem: A while ago the synchronisation between the deconz objects and the objects in iobroker started to become unreliable. Some items were automatically put into folders from other items (i.e. item of sensor X in folder of sensor Y) and the titles seemed to be mixed up (i.e. sensor name in deconz "officePresence" was named "presence 45" in iobroker ) while others got their names from deconz correctly. Since a few updates and changes it seems, new objects in deconz are not recognized by the adapter anymore. And they do not show up in the devices view of the adapter nor in the objects view under deconz. While at the same time in openHAB everything runs fine and new items are recognized as usual. Also, the objects are working fine, which are already set up in ioBroker. Sensor presence is detected, light status is correct, ... so, the connection does not seem to be the problem. But no new objects. Is there a limitation to the amount of objects? I read somewhere, that the latest versions and betas from deconz are not supported by the adapter. Is that the cause to my problem or do I need to re-iniate something?
Iam using ioBroker 3.1.6, ioBroker.deconz Adapter version 1.3.11, deConz Version: 2.5.86 (headless) + latest conBEEII firmware, API Version: 1.16.0

**Comment by Jey-Cee** [comment]
I have no idea whats the problem on your side. 
The first thing you can do is to check the log in debug mode of the adapter. There you can see what messages from deConz coming in and can check if they are correct.

I saw a lot of issues where the problem was the data from deConz are missing or the ids of deleted devices are given to new devices and makes trouble.

**Comment by psanwald** [comment]
Thank you, for your comment, Jey-Cee. I can confirm, that the problem has got to do with given ids from deconz. After deleting an item there, the id seems to be recycled and released for new items. So i.e. item#2-lightbulb becomes item#2-heatcontrol. That puts iobroker objects into deep chaos. I figured that out and could find ALL corresponding objects within this chaos. All in all the error is definetly not, that the objects are not synchronized completely. They are and you did a perfect job. Also, I think I figured out, why it works in openHab nevertheless. They use the name as identifier - not the id. At least I could reproduce that with changing an object and give it the same name. So, the only problem remains is the recycled ids from deconz. And as you wrote, that is in responsibility of the deconz developers.
